src/main/java/frc/robot/Robot.java

```diff
@@ -192,7 +203,15 @@ public static Robot win() {
     }
 
     public void shoot() {
-        turretSubsystem.setAzimuth(visionSubsystem.getAngle());
+        //TODO Calculate azimuth using distance, x, y
+
+        Vision.Coordinates coords = vision.getCoordinates();
+
+        new CoordinateDistance(coords).distance();
+        coords.getVerticalOffset();
+        coords.getHorizontalOffset();
+
+        turretSubsystem.setAzimuth(0);
```

Comment:
We need the vision subsystem to tell us what angle to rotate to, in degrees. Can you update this to do that?

Reply:
Yeah sure just gotta add some math stuff